packages/testing/examples/connectContractToSigner.test.ts

```diff
@@ -0,0 +1,45 @@
+import { ERC20Interface, connectContractToSigner, useEthers } from '@usedapp/core'
+import chai, { expect } from 'chai'
+import { MockProvider, solidity } from 'ethereum-waffle'
+import { Contract } from 'ethers'
+import { renderWeb3Hook } from '../src'
+
+chai.use(solidity)
+
+describe.only('useContractFunction', () => {
```

Comment:
`.only` 
Tip: better to use `yarn test -g 'useContractFunction'` , than `.only` prefix